### ISSUE TITLE
Incorrect spelling in settings

### ISSUE THREAD

**Issue by ahadley1124** [lucanaef/PGPro#112]
The License page is spelled correctly when clicked on but the button is spelled i correctly in the beta. ![image](https://user-images.githubusercontent.com/49847299/230695833-be602c4c-b4f7-4ba4-a826-14cc65ac3dd5.jpeg)

**Comment by lucanaef** [comment]
Thanks!

**Comment by ahadley1124** [comment]
@lucanaef The spelling still is incorrect. The proper spelling should be "license". Thanks.

**Comment by lucanaef** [comment]
Shouldn't it be plural? In which case “Licences” seems to be a valid spelling [1].

[1] https://simple.m.wiktionary.org/wiki/licences

**Comment by ahadley1124** [comment]
You are correct. I do see the change when clicking on the link but the button is still spelled incorrectly. ![image](https://user-images.githubusercontent.com/49847299/233160463-ef7a60cb-65d8-4a8e-8322-a2a89b82603f.png)![image](https://user-images.githubusercontent.com/49847299/233160492-cd733fa3-673a-4841-b60b-17394364638f.png)
